vic/drivers/cesm/src/display_current_settings.c

```diff
@@ -398,11 +398,11 @@ display_current_settings(int mode)
         fprintf(LOG_DEST, "STATEYEAR\t\t%d\n", global_param.stateyear);
         fprintf(LOG_DEST, "STATEMONTH\t\t%d\n", global_param.statemonth);
         fprintf(LOG_DEST, "STATEDAY\t\t%d\n", global_param.stateday);
-        if (options.BINARY_STATE_FILE) {
-            fprintf(LOG_DEST, "BINARY_STATE_FILE\tTRUE\n");
+        if (options.STATE_FORMAT) {
+            fprintf(LOG_DEST, "STATE_FORMAT\tTRUE\n");
```

Comment:
I may be getting lost in the drivers and options, but I thought that `STATE_FORMAT` had to be a specific format `BINARY | ASCII | etc` rather than `TRUE | FALSE` -- same for following line.